### DISCUSSION TITLE
Running hms-mqtt-publisher on Portainer

### DISCUSSION THREAD

**Discussion by Blacerrr** [DennisOSRM/hms-mqtt-publisher#100] [Q&A]
Hi everyone, 

Im really lost and don't know what to look out for. For me it seems like that running hms-mqtt-publisher with Docker would be the easiest solution. Unfortunatly I can't get it to run. It always exits instantly after I created the container. I added the image and also added all the requested environmental variables as you can tell from the screenshot. Is there anything else super basic that I'm missing which would cause this container not to start at all? But it seems like that that this would be the easiest solution for me to run this without any manual compilation at least from my very basic understanding so I don't want to give up on it just yet.

Edit: Tried running on Docker desktop without success.

The log:
`[2024-02-24T16:47:21Z INFO  hms_mqtt_publish] loading configuration from /config.toml
thread 'main' panicked at src/main.rs:55:52:
toml config unparsable: Error { inner: Error { inner: TomlError { message: "invalid string\nexpected `\"`, `'`", original: Some("inverter_host = \"\" \n\n[home_assistant] \nhost = \"\"\nusername = \"\"\npassword = \"\"\nport = \n\n"), keys: [], span: Some(84..85) } } }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
`

Thanks kindly. All help is much appreciated

![Screenshot 2024-02-24 170022](https://github.com/DennisOSRM/hms-mqtt-publisher/assets/13471958/71dd6337-82cf-491f-8d43-a805c2264d6e)

**Comment by Diemo79** [comment]
Hi,

i have the same issue. Did you fixed the problem in the meantime?

Thanks a lot...

**Comment by Blacerrr** [comment]
Don't bother with it. It is broken. If you use HA download it from HACS.
Less PITA.

Am Fr., 26. Juli 2024 um 22:04 Uhr schrieb Diemo79 ***@***.***
